home.html

```diff
@@ -0,0 +1,118 @@
+</head>
+<body class="d-flex flex-column" id="home">
+  <header class="menu">
+    <nav class="menu-btn">
+      <div class="menu-btn__burger"></div>
+    </nav>
+    <nav>
+      <section class="menu-d-items" id="dmenu-item">
+        <section class="menu-logo menu-desktop">
+          <a href="home.html"><img src="img\logo.png" alt="logo"></a>
+        </section>
+        <ul class="item">
+          <li class="nav-item-btn"><a href="about.html">About</a></li>
+          <li class="nav-item-btn"><a href="#mainProgram">Program</a></li>
+          <li class="nav-item-btn"><a href="#ticket">Ticket</a></li>
+          <li class="nav-item-btn"><a href="#performerSection">Performers</a></li>
+        </ul>
+      </section>
+    </nav>
+  </header>
+  <section class="intro">
+    <h2 class="h2-orange-title">"Classical Music"</h2>
+    <h1>FREDERIC CHOPIN <br>CONCERT 2023</h1>
+    <section class="detail">
+      <p>Lorem ipsum dolor sit amet, consectetur 
+        adipisicing elit. Tenetur, reprehenderit.</p>
+    </section>
+    <h2 class="date">2023.03.18(SAT) ~ 19(SUN)</h2>
+    <p class="location">@ Warsaw, Poland</p>
+  </section>
+  <div id="mainProgram" class="moveUp"></div>
+  <main class="dflex-column py-5">
+    <h2 class="white-title">Main program</h2>
+    <div class="bar"></div>
+    <ul class="program">
+      <li>
+        <img src="img\G key.png" alt="pic">
+        <h3 class="program-titles">Opening</h3>
+        <p>Opening of the event with the presentation of the performers Lorem ipsum 
+          dolor sit amet consectetur adipisicing.
+        </p>
+      </li>
+      <li>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">Nocture op</h3>
+        <p>A devastating melody that lures you in at pianissimo, 
+          and one most likely recognised for its use in 
+          Roman Polanski’s The Pianist.
+        </p>
+      </li>
+      <li>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">Fantaisie-Impromptu</h3>
+        <p>The posthumously published Fantaisie-Impromptu is one 
+          of Chopin’s most ostentatiously improvised works, and a 
+          fantastic show piece for the piano.
+        </p>
+      </li>
+      <li>
+        <img src="img\piece.png" alt="pic">
+        <h3 class="program-titles">24 Preludes</h3>
+        <p>Each prelude contains a unique idea or feeling, from 
+          the pianistic precipitation in ‘Raindrop’ to the haunting 
+          E minor prelude, which was played at Chopin’s own funeral.
+        </p>
+      </li>
+      <li>
+        <img src="img\F key.png" alt="pic">
+        <h3 class="program-titles">Closing</h3>
+        <p>Closing and presentation of the band, Lorem ipsum dolor 
+          sit amet consectetur adipisicing elit. Ut, blanditiis!
+        </p>
+      </li>
+    </ul>
```

Comment:
I kindly suggest you use the "ARTICLE" semantic tag (instead of UL/LI), this is ideal for repetitive content like this